dapps/marketplace/src/pages/growth/Action.js

```diff
@@ -186,91 +191,93 @@ function Action(props) {
       }`}
     >
-      <div className="col-1 pr-0 pl-0 d-flex justify-content-center">
-        <div className="image-holder mt-auto mb-auto">
-          {type !== 'ListingIdPurchased' && (
+      <div className="d-flex action-main">
+        <div className="col-1 pr-0 pl-0 d-flex justify-content-center align-items-center">
+          {type === 'ListingIdPurchased' ? (
+            <img className={type.toLowerCase()} src={foregroundImgSrc} />
+          ) : (
             <Fragment>
               <img className="background" src={backgroundImgSrc} />
               <img className={type.toLowerCase()} src={foregroundImgSrc} />
             </Fragment>
           )}
-          {type === 'ListingIdPurchased' && (
-            <Fragment>
-              <img className={type.toLowerCase()} src={foregroundImgSrc} />
-            </Fragment>
-          )}
         </div>
-      </div>
-      <div className={`d-flex flex-column justify-content-center col-6`}>
-        <div className="title">{title}</div>
-        <div className="details">
-          {detailsKey && (
+        <div className={`d-flex flex-column justify-content-center col-6`}>
+          <div className="title">{title}</div>
+          {actionLocked && !isMobile && unlockConditions.length > 0 && (
             <Fragment>
-              <fbt desc="growth">
-                <fbt:enum enum-range={GrowthEnum} value={detailsKey} />
-              </fbt>
+              <div className="requirement pr-2 d-flex align-items-center ">
+                {unlockConditionText}
+              </div>
             </Fragment>
           )}
         </div>
-        {actionLocked && !isMobile && unlockConditions.length > 0 && (
-          <Fragment>
-            <div className="requirement pr-2 d-flex align-items-center ">
-              {unlockConditionText}
-            </div>
-          </Fragment>
-        )}
-      </div>
-      <div className="col-5 d-flex align-items-center justify-content-end">
-        {showReferralPending && (
-          <div className="d-flex flex-column">
-            {renderReward(rewardPending.amount)}
-            <div className="sub-text ml-4">
-              <fbt desc="RewardActions.pending">Pending</fbt>
-            </div>
-          </div>
-        )}
-        {showReferralEarned && (
-          <div className="d-flex flex-column">
-            {renderReward(rewardEarned.amount)}
-            <div className="d-center sub-text ml-4">
-              <fbt desc="RewardActions.earned">Earned</fbt>
+        <div className="col-5 d-flex align-items-center justify-content-between">
+          <a
+            href="#"
+            className={`toggle-details${detailsKey ? '' : ' invisible'} mr-3`}
+            onClick={e => {
+              e.preventDefault()
+              e.stopPropagation()
+              toggleDetails(!detailsToggled)
+            }}
+          >
+            {detailsToggled ? (
+              <fbt desc="RewardActions.hideDetails">Hide Details</fbt>
```

Comment:
nit: run npm translate to pick up the new strings "Hide Details" and "View Details"